### PR TITLE
ide-usage.md: add steps to disable JSON validation for node_modules folder in Eclipse #7326

### LINKED ISSUE #7326: ide-usage.md: add steps to disable JSON validation for node_modules folder in Eclipse
JSON validation is present in newer versions of Eclipse which is validating JSON files in `node_modules` folder. Some files have been reported to cause errors #7310 . We must disable JSON validation for `node_modules` folder so that these errors are suppressed. 

Add steps for disabling JSON validation in `ide-usage.md`, under `Validation`.

### PR DESCRIPTION
**Pull request by Andrew-Lev**
Fixes #7326

Added instruction text to ide-usage.md file for disabling JSON validation for node_modules folder when using Eclipse ide.

### CODE REVIEW

File: docs/ide-usage.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ The following plugins are needed:
    * Validation:
      * We do not validate HTML, JSP, and XML. `Validation` → uncheck the `Build` option for `HTML Syntax Validator`, `JSP Content Validator`, `JSP Syntax Validator`, and `XML Validator`.
      * Disable JavaScript validation for `node_modules` folder. `Validation` → click the `...` settings button for `JavaScript Validation` → if `Exclude Group` is not already in the list then click `Add Exclude Group...` → `Exclude Group` → `Add Rule...` → `Folder or file name` → `Next` → `Browse Folder...` → navigate to the `node_modules` folder and confirm → `Finish`.
+	 * Disable JSON validation for `node_modules` folder. `Validation` → click the `...` settings button for `JSON Validator` → if `Exclude Group` is not already in the list then click `Add Exclude Group...` → `Exclude Group` → `Add Rule...` → `Folder or file name` → `Next` → `Browse Folder...` → navigate to the `node_modules` folder and confirm → `Finish`.
```
Comment (VamsiSangam) [inline docs/ide-usage.md]: > settings button for `JSON Validator`**. If**

> is not already in the list, then click `Add Exclude Group...`. **In the list, click** `Exclude Group`

**Comment by Andrew-Lev** [comment]
Ok sure, I used the same wording as was present for the disable JavaScript validation steps.  Do you want me to update the text for that one also?

**Comment by VamsiSangam** [comment]
On the second thought, I think the JavaScript validation steps are written more accurately. Please revert the changes. Apologies for the confusion.

**Comment by Andrew-Lev** [comment]
Ok.

**Comment by VamsiSangam** [comment]
@leeyimin reminder

**Comment by whipermr5** [comment]
@Andrew-Lev Do take note of our [PR naming conventions](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr) for future PRs; I've helped you fix it this time :)

**Pull request merged**